packages/trader/src/AppV2/Hooks/useOrderDetails.ts

```diff
@@ -0,0 +1,163 @@
+import {
+    CONTRACT_TYPES,
+    TContractInfo,
+    addComma,
+    getDurationPeriod,
+    getDurationTime,
+    getDurationUnitText,
+    getGrowthRatePercentage,
+    isAccumulatorContract,
+    isResetContract,
+} from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { getBarrierValue } from 'App/Components/Elements/PositionsDrawer/helpers';
+
+// Contains all key values that are used more than once in different transform objects
+const getCommonFields = (data: TContractInfo) => {
+    const { tick_count, tick_passed, contract_type } = data;
+    const ticks_label = Number(tick_count) < 2 ? localize('tick') : localize('ticks');
+    const ticks_duration_text = isAccumulatorContract(contract_type)
+        ? `${tick_passed}/${tick_count} ${localize('ticks')}`
+        : `${tick_count} ${ticks_label}`;
+
+    return {
+        [localize('Reference ID')]: [
+            data.transaction_ids?.buy ? `${data.transaction_ids.buy} (Buy)` : '',
+            data.transaction_ids?.sell ? `${data.transaction_ids.sell} (Sell)` : '',
+        ],
+        [localize('Stake')]: data.buy_price ? `${data.buy_price.toFixed(2)} ${data.currency}` : '',
+        [localize('Duration')]:
+            Number(tick_count) > 0
+                ? ticks_duration_text
+                : `${getDurationTime(data) ?? ''} ${getDurationUnitText(getDurationPeriod(data)) ?? ''}`,
+        [localize('Payout per point')]: data.display_number_of_contracts ? `${data.display_number_of_contracts}` : '',
+    };
+};
+
+// For Multiplier
+const transformMultiplierData = (data: TContractInfo) => {
+    const commonFields = getCommonFields(data);
+    return {
+        [localize('Reference ID')]: commonFields[localize('Reference ID')],
+        [localize('Multiplier')]: data.multiplier ? `x${data.multiplier}` : '',
```

Comment:
Can we add a V2 version of `getCardLabels()` config in /Utils to include all these new labels?
```
export const getCardLabels = () =>
    ({
        COMMISSION: localize('Commission'),
        MULTIPLIER: localize('Multiplier'),
        NOT_SET: localize('Not set'),
        STAKE: localize('Stake'),
        STOP_OUT_LEVEL: localize('Stop out level'),
        REFERENCE_ID: localize('Reference ID'),
// ... so on
```
so that we can use them in this file?